etcd/fs/fs_test.go

```diff
@@ -245,9 +259,8 @@ func TestFs4(t *testing.T) {
 		DataPrefix: etcdfs.DefaultDataPrefix,
 	}
 
-	etcdFs.Mkdir("/tmp", umask)
```

Comment:
why did you -1 on depth here?

Reply:
There's no need for `/tmp`, but it's not a bad idea to test it with a deeper copy.